server/routerlicious/package.json

```diff
@@ -75,7 +75,9 @@
     ],
     "temp-directory": "nyc/.nyc_output"
   },
-  "dependencies": {},
+  "dependencies": {
```

Comment:
In server, the dependency on `async<3.2.2` seems to come transitively from a dependency on `winston@3.3.3`, which updated its dependency to `async@^3.2.3` on 3.4.0, so we could update `winston` and that should resolve the CVE.